pom.xml

```diff
@@ -160,6 +177,12 @@
         <artifactId>maven-verifier</artifactId>
         <version>${version.org.apache.maven.shared}</version>
       </dependency>
+
+      <dependency>
+        <groupId>org.eclipse.jgit</groupId>
+        <artifactId>org.eclipse.jgit</artifactId>
+        <version>4.11.0.201803080745-r</version>
```

Comment:
Just a nitpick, please put the version into the properties so they all are in one place.

Reply:
Good catch, will improve it.